src/main/resources/assets/farmersdelight/lang/ko_kr.json

```diff
@@ -67,22 +72,24 @@
   "item.farmersdelight.minced_beef": "다진 소고기",
   "item.farmersdelight.beef_patty": "소고기 패티",
   "item.farmersdelight.chicken_cuts": "익히지 않은 닭고기 살점",
-  "item.farmersdelight.cooked_chicken_cuts": "구운 닭고기 살점",
+  "item.farmersdelight.cooked_chicken_cuts": "익힌 닭고기 살점",
   "item.farmersdelight.bacon": "베이컨",
   "item.farmersdelight.cooked_bacon": "구운 베이컨",
   "item.farmersdelight.cod_slice": "생대구 살점",
   "item.farmersdelight.cooked_cod_slice": "익힌 대구 살점",
   "item.farmersdelight.salmon_slice": "생연어 살점",
   "item.farmersdelight.cooked_salmon_slice": "익힌 연어 살점",
+  "item.farmersdelight.mutton_chops": "익히지 않은 양고기 살점",
+  "item.farmersdelight.cooked_mutton_chops": "익힌 양고기 살점",
   "item.farmersdelight.ham": "햄",
   "item.farmersdelight.smoked_ham": "훈제 햄",
-
+  
   "item.farmersdelight.pie_crust": "파이 껍질",
   "item.farmersdelight.cake_slice": "케이크 조각",
   "item.farmersdelight.apple_pie_slice": "사과 파이 조각",
   "item.farmersdelight.sweet_berry_cheesecake_slice": "달콤한 열매 치즈케이크 조각",
   "item.farmersdelight.chocolate_pie_slice": "초콜릿 파이 조각",
-  "item.farmersdelight.fruit_salad": "과일 샐러드",
```

Comment:
From Google Translate, it seems the previous would be correct, with the new one being just "fruit". Is this accurate?

Reply:
Oof. I accidentally deleted salad part 😅